.ibm/pipelines/utils.sh

```diff
@@ -549,7 +549,9 @@ run_tests() {
   cp -a "/tmp/${LOGFILE}.html" "${ARTIFACT_DIR}/${project}"
   cp -a /tmp/backstage-showcase/e2e-tests/playwright-report/* "${ARTIFACT_DIR}/${project}"
 
-  droute_send "${release_name}" "${project}"
+  if [[ "${OPENSHIFT_CI}" == "true" ]]; then
```

Comment:
just a suggestion, this method is becoming big and big. does this makes sense to move this condition inside the droute_send() and something like
if [[ "${OPENSHIFT_CI}" != "true" ]]; then
    return 0
fi

Reply:
Yes, we can do that 👍